covid/extract.py

```diff
@@ -44,3 +45,73 @@ def get_state_abbreviations_to_names():
         abbreviations = json.load(state_abbreviations_file)
 
     return abbreviations
+
+
+def power_bi_extractor(response):
```

Comment:
So nice that you've made a clean reusable method out of this!

Reply:
It's too ugly to repeat!